src/components/request-content-modal/request-content.modal.tsx

```diff
@@ -20,7 +20,7 @@ const RequestContentModal: React.FunctionComponent<RequestContentModalProps> =
             const contentRequest: ContentRequest = {
                 topic,
                 description,
-                email,
+                email: email || null, // Convert empty string to null.
```

Comment:
are we saying we don't need email to request content ?

Reply:
That's correct, Amy had [a PR that validated input](https://github.com/mongodb/devcenter/pull/388), and it was decided by Rachelle that email should be optional.